app/src/main/java/com/abhijitvalluri/android/fitnotifications/AppSettingsActivity.java

```diff
@@ -116,7 +118,8 @@ public void onClick(View v) {
                                                                            mStartTimeMinute,
                                                                            mStopTimeHour,
                                                                            mStopTimeMinute,
-                                                                           R.string.start_time_heading,
+                                                                           stopTimeFormatted,
```

Comment:
You must also fix lines 197 and 202 so that the time is set in the correct format once returned from TimePickerFragment.